tests/test_board.py

```diff
@@ -1,29 +1,30 @@
-from app.board import Board
+from killer_queen.board import Board
+
 
 class TestBoard:
-  def test_placing_a_piece(self):
-    b = Board(4)
-    b.place(0, 1)
-    assert b.positions[0][1] == True
+    def test_placing_a_piece(self):
+        b = Board(4)
+        b.place(0, 1)
+        assert b.positions[0][1] is True
 
-  def test_removing_a_piece(self):
-    b = Board(4)
-    b.place(0, 1)
-    b.remove(0, 1)
-    assert b.positions[0][1] == False
+    def test_removing_a_piece(self):
+        b = Board(4)
+        b.place(0, 1)
+        b.remove(0, 1)
+        assert b.positions[0][1] is False
 
-  def test_has_piece(self):
-    b = Board(4)
-    b.place(0, 2)
-    assert b.has_piece(0, 2)
+    def test_has_piece(self):
+        b = Board(4)
+        b.place(0, 2)
+        assert b.has_piece(0, 2)
 
-  def test_board_does_not_have_piece(self):
-    b = Board(4)
-    assert b.has_piece(0, 1) == False
+    def test_board_does_not_have_piece(self):
+        b = Board(4)
+        assert b.has_piece(0, 1) is False
```

Comment:
`value is False` and `value is None` is the pythonic practice over `value == False` or `value == None`